osu.Server.Spectator/Hubs/Multiplayer/MultiplayerHub.cs

```diff
@@ -720,8 +731,8 @@ private async Task removeDatabaseUser(MultiplayerRoom room, MultiplayerRoomUser
 
         protected override async Task CleanUpState(MultiplayerClientState state)
         {
-            await leaveRoom(state, true);
             await base.CleanUpState(state);
+            await leaveRoom(state, true);
```

Comment:
Meaningless change because the base implementation is empty. I just thought I'd make this consistent with other implementations, seeing as `leaveRoom` can throw an exception.